PreProcessing/identify_kinematic_chains.m

```diff
@@ -346,17 +346,28 @@
     joints_chain = [];
```

Comment:
Can you add a function header here?

Reply:
Added header that includes the structure of the variables.

Reply:
Thanks!